x-pack/legacy/plugins/reporting/export_types/printable_pdf/server/lib/generate_pdf.js

```diff
@@ -35,7 +35,6 @@ function generatePdfObservableFn(server) {
   const urlScreenshotsObservable = (urls, conditionalHeaders, layout, browserTimezone) => {
     return Rx.from(urls).pipe(
       mergeMap(url => screenshotsObservable(url, conditionalHeaders, layout, browserTimezone),
```

Comment:
This didn't do anything.

Reply:
I see this part of the change was not backported to 7.x: https://github.com/elastic/kibana/pull/41066/files - unfortunately I am getting a merge conflict for my prettier changes on this. Thanks for adding this comment: looks to me that it's ok to resolve my conflict by doing the same removal. :)